apps/e2e/helpers/table.ts

```diff
@@ -148,6 +148,15 @@ const stringFieldConstructor =
 			expect(row.locator(`[data-property="${name}"]`)).toHaveText(transformDisplay(want.toString()), { timeout: assertionTimeout, ...opts })
 	});
 
+const priceFieldConstructor: FieldConstructor<"price"> = (row) => ({
+	assert: (want: string | number, opts) =>
+		typeof want === "string"
+			? expect(row.locator(`[data-property="price"]`)).toHaveText(want, { timeout: assertionTimeout, ...opts })
+			: expect(row.locator(`[data-property="price"]`)).toHaveText(`€${want.toFixed(2)}`, {
+					timeout: assertionTimeout,
+					...opts
+			  })
+});
```

Comment:
Continuing from the comment in the price display element(s), let's have the price field asserter ☝️ accept a price that is either:
- string - used to only match the price (regardless of price being discounted
- object with shape:
  - price - string
  - discount (optional) - number
  - discounted price (optional) - string

That way we have a more fine grained control over the assertion:
- if string - only the full price is asserted (regardless of discount and discounted price
- if object - more strict approach:
  - every field is asserted exactly as specified
  - if optional fields are present - they're expected to not be there